cleaning up of some v1 applications

muhammad-ali-e opened the pull request:
## What

- Removed specific v1 applications as part of an ongoing cleanup process.
  -  account
  - tenant_account
  - adapter_processor
  - api
  - connecter
  - connector_auth
  - notification
  - pipeline
  - platform_settings
  - tool_instance
  - usage

## Why

- This is part of a phased approach to deprecate v1 applications. Incremental removal minimizes risks and ensures better maintainability.

## How

-  Deleted the above-listed v1 applications. Remaining v1 applications will be addressed in future PRs.

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No: These applications are not referenced or imported elsewhere, so their removal will not affect any existing functionality.

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Screenshots

## Checklist

I have read and understood the [Contribution Guidelines]().

chandrasekharan-zipstack reviewed (commented):
@muhammad-ali-e you might want to [check this](https://docs.djangoproject.com/en/5.1/howto/delete-app/) on deleting apps. We need to delete the models and generate migrations to ensure that those tables also get removed. Do you think that might not be possible now since we already removed v1 apps from `INSTALLED_APPS`?

muhammad-ali-e commented:
> @muhammad-ali-e you might want to [check this](https://docs.djangoproject.com/en/5.1/howto/delete-app/) on deleting apps. We need to delete the models and generate migrations to ensure that those tables also get removed. Do you think that might not be possible now since we already removed v1 apps from `INSTALLED_APPS`?

Yes, It  not be possible because of we removed it from `INSTALLED_APPS`.  And the second thing is we don't do this because of it might be consider as the Data backup  for sometime. We can remove the data by Drop and Recreate the Schema .
